src/stores/widgets/SpectralLineQueryWidgetStore.ts

```diff
@@ -558,15 +557,8 @@ export class SpectralLineQueryWidgetStore {
     };
 
     @action pingSplatalogue = async () => {
-        try {
-            this.splataloguePingStatus = SplataloguePingStatus.Checking;
```

Comment:
only one minor question: do we need to keep this line for reseting status?

Reply:
good point. I've added it back, but also switch from an `@action` to a `flow` (see here for [reasoning](https://mobx.js.org/actions.html#using-flow-instead-of-async--await-). We should be doing this throughout the codebase to avoid modifying observables outside of actions when using async functions).